src/contentfulImport/command.js

```diff
@@ -28,29 +29,37 @@ export default async (contentfulToken, contentfulSpaceId, datoCmsToken) => {
 
   const fieldsMapping = await createFields({ itemTypes, datoClient, contentfulData });
 
-  const contentfulRecordMap = await createRecords({
-    itemTypes,
-    fieldsMapping,
-    datoClient,
-    contentfulData,
-  });
+  if (!skipContent) {
+    const contentfulRecordMap = await createRecords({
+      itemTypes,
+      fieldsMapping,
+      datoClient,
+      contentfulData,
+    });
 
-  await createUploads({
-    fieldsMapping,
-    itemTypes,
-    datoClient,
-    contentfulData,
-    contentfulRecordMap,
-  });
+    await createUploads({
+      fieldsMapping,
+      itemTypes,
+      datoClient,
+      contentfulData,
+      contentfulRecordMap,
+    });
+
+    await linkRecords({
+      fieldsMapping,
+      datoClient,
+      contentfulData,
+      contentfulRecordMap,
+    });
+
+    await publishRecords({
```

Comment:
@ullmark what I would do is to `publishRecords` before `linkRecords` as if records are not published it won't pass the validations on Dato

Reply:
@matjack1 So, you were correct, and I were correct so I did another update :) Right now it works like this;

1. `createRecords` now also exports a list of dato-ids that should be published.
2. `publishRecords` is executed before link to make sure linked records are properly published.
3. `linkRecords` now only calls update when needed and also exports a list of ids that needs to be "re"-published.
4. `publishRecords` publishes the items that had their link fields set. 

(sorry, haven't used this change request/start a review thing of github just yet, so I'll add this as a regular comment. Also, my commits are starting to stack up, If you want me to rebase, let me know).